CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.20.1
+
+* `DEPS`: update to `@bpmn-io/properties-panel@3.23.0`

```

Comment:
An entry for [the issue](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/713) is missing.